save-to-webm/main.go

```diff
@@ -51,53 +58,47 @@ func (s *webmSaver) Close() {
 	}
 }
 func (s *webmSaver) PushOpus(rtpPacket *rtp.Packet) {
+	s.audioBuilder.Push(rtpPacket)
+
+	sample := s.audioBuilder.Pop()
+	if sample == nil {
+		return
+	}
 	if s.audioWriter != nil {
-		if s.audioStartTimestamp == 0 {
-			s.audioStartTimestamp = rtpPacket.Timestamp
-		}
-		if rtpPacket.Timestamp < s.audioStartTimestamp {
-			panic("RTP Timestamp overflow. Please add proper timestamp processor to continuously save stream!")
-		}
-		t := (rtpPacket.Timestamp - s.audioStartTimestamp) / 48
+		s.audioTimestamp += sample.Samples
```

Comment:
It would be better to take duration of lost frames into account in real applications.
(related: https://github.com/pion/webrtc/pull/931)